pkgs/development/libraries/nanoflann/generic-cmake.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
```

Comment:
Too much work to keep a special case.

An overrideAttrs, or even a mere copy-paste, is better suited for this.

Reply:
Good idea, thanks for the pointer!